ui/packages/ui/src/main/webapp/components/configuration-item/configuration-item.hbs

```diff
@@ -12,11 +12,11 @@
  **/
  --}}
 <div>
-    <a href='#{{uuid}}' class='editLink' data-toggle="modal" data-backdrop="static" data-keyboard="false">
+    <a href='#{{cid}}' class='editLink' data-toggle="modal" data-backdrop="static" data-keyboard="false">
```

Comment:
What is the `cid`?

Reply:
```A special property of models, the cid or client id is a unique identifier automatically assigned to all models when they're first created ```

We've swapped this over to something we'll know will be unique instead of this uuid.